go/signedexchange/structuredheader/parser.go

```diff
@@ -96,6 +116,31 @@ func (p *parser) parseKey() (Key, error) {
 	}
 	return Key(p.getString(i)), nil
 }
+// https://tools.ietf.org/html/draft-ietf-httpbis-header-structure-09#section-4.2.4
+func (p *parser) parseListOfLists() (ListOfLists, error) {
+	var top_list ListOfLists
```

Comment:
Style nit: https://golang.org/doc/effective_go.html#mixed-caps

Note to self: I was going to comment `innerList := []Item{}`, but found https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices

Reply:
Done